R/mod-upload-documentation.R

```diff
@@ -151,6 +151,15 @@ upload_documents_server <- function(input, output, session,
         if (study_name() == "") {
           stop("Please enter study name.")
         }
+        # Check if study name has inappropriate characters
+        temp_string <- stringr::str_replace_all(
```

Comment:
Could you add stringr under Imports in the DESCRIPTION file?

Reply:
Done.

Side note: the commit for this change had the pre-commit hooks turned off since 'use-tidy-description' failed every time. It doesn't fail if I run `usethis::use_tidy_description()' from RStudio within the project. The only thing I can think of is the fact that it prints output saying that it's changing the active project to dccvalidator (it is the active project so that's odd regardless). The output might be being flagged as an error.